src/bpftrace.h

```diff
@@ -214,12 +214,13 @@ class BPFtrace
       int pid,
       bool file_activation);
   int setup_perf_events();
-  void poll_perf_events(int epollfd, bool drain = false);
+  void poll_perf_events(bool drain = false, int timeout_ms = 100);
```

Comment:
Any reason to expose `timeout_ms`? Doesn't look like there's any callers that set !100. Maybe make it a constant instead?

Reply:
Made it a constant in this PR.

It could be useful to have it be a parameter that is controllable via arguments. I may put out a separate PR for exposing the timeout; we can discuss the merits of it at that time. For now, better to keep PRs focused :)

Reply:
If this were a library, I'd agree. But for internal use only, we should only expose APIs that are actively used. Adding theoretically useful stuff leads to cruft (see YAGNI)